frontend/src/components/Election/Admin/EditElectionRoll.tsx

```diff
@@ -36,6 +38,14 @@ const EditElectionRoll = ({ roll, onClose, fetchRolls, id, permissions }) => {
     }
     const onSendInvite = async () => {
         if (!await sendInvite.makeRequest()) { return }
+
+        setSnack({
```

Comment:
The useFetch hook actually has this snack feature built in, you can set an optional success message on a successful request. I never got around to using it for anything.

Reply:
Good point, updated!